src/common/locations.ts

```diff
@@ -100,10 +102,19 @@ export function findCountyByFips(fips: string) {
   return undefined;
 }
 
-export function findStateByFips(fips: string): State {
-  const state = US_STATE_DATASET.state_dataset.find(
+function _findStateByFips(fips: string): State | undefined {
+  return US_STATE_DATASET.state_dataset.find(
     state => state.state_fips_code === fips,
   );
+}
+
+export function isStateFips(fips: string): boolean {
+  const state = _findStateByFips(fips);
+  return !!state;
+}
```

Comment:
Sometimes we use `isStateFips` to differentiate state and county fips codes, so I wouldn't like to run the assertion in that case.